src/parcsr_ls/par_amg_solveT.c

```diff
@@ -703,10 +703,11 @@ HYPRE_Int  hypre_BoomerAMGRelaxT( hypre_ParCSRMatrix *A,
           *  Generate CSR matrix from ParCSRMatrix A
           *-----------------------------------------------------------------*/
 
+         A_CSR = hypre_ParCSRMatrixToCSRMatrixAll(A);
+         f_vector = hypre_ParVectorToVectorAll(f);
+         /* WM: If statement was in the wrong place? Need all procs to participate in comm above */
```

Comment:
It seems that the change is correct. But I am not 100% sure.

Reply:
Yeah, this fixed a bug for me and also makes sense to me.